cms/settings/base.py

```diff
@@ -345,8 +345,10 @@
 STATICFILES_DIRS = [
     # "static_compiled" is a folder used by the front-end tooling
     # to output compiled static assets.
+    # "static" is a folder used for any static assets that are not compiled.
     (PROJECT_DIR / "jinja2" / "assets"),
     (PROJECT_DIR / "static_compiled"),
+    (PROJECT_DIR / "core" / "static"),
```

Comment:
This shouldn't be necessary, as Django will use the app dirs finder to find the static files.

Reply:
Thanks, removed.